api/repositories/buildpack_repository_test.go

```diff
@@ -57,11 +80,34 @@ var _ = Describe("BuildpackRepository", func() {
 					}),
 				))
 			})
+
+			Describe("ordering", func() {
```

Comment:
In order to improve test readability and reduce the nesting level, how about setting the `OrderBy` field on the message in the `Describe("list"` 's `BeforeEach` and have the `It("sorts the buildpacks` on the same level as `It("returns all buildpacks"`?

Reply:
I've tried to follow your suggestion. Is it now what you've expected?